### PR TITLE
Add doc about Proactive installation using docker containers

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -149,6 +149,345 @@ Once the server is up, ProActive Web Portals are available at `\http://localhost
...
+
+==== How to install ProActive using Docker Containers
+ProActive could be easily deployed and started within a fully containerized environment. For this purpose, we provide a ready-to-use Docker Compose configuration that enables you running ProActive Workflows and Scheduling as a containerized service.
+It starts the following services:
+
+*  Proactive server container
+*  Proactive node container
+*  Proactive database container (Postgres or Mysql). The database container is not started when using our embedded HSQL database.
+
+The Docker Compose configuration pulls the Proactive server, node, and database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
```
Comment (marounkoussaifi) [inline src/docs/admin/ProActiveAdminGuide.adoc:161]: "," after "node" and before "and" is not required

Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc:161]: It is https://www.grammarly.com/blog/comma-before-and/

**Review by marounkoussaifi — APPROVED** [review]

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -149,6 +149,345 @@ Once the server is up, ProActive Web Portals are available at `\http://localhost
...
+  node-previous:
+  proactive-data:
+
+networks:
+  pa-network:
+----
+
+===== Start Proactive using Docker Compose
+
+Login into the Private Activeeon Regitry
```
[the remark this answers is not in the document]
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc]: done thx

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -149,6 +149,345 @@ Once the server is up, ProActive Web Portals are available at `\http://localhost
...
+DB_SCHEDULER_PASS=change_me
+DB_CATALOG_PASS=change_me
+DB_PCA_PASS=change_me
+DB_NOTIFICATION_PASS=change_me
+
+----
+
+Once, you create and configure the `.env` file, you can select one of the three following Docker Compose configurations to start the Proactive server and node containers.
+
+For instance, we provided support to three database systems, namely HSQLDB, Postgres, and MySQL.
```
Comment (Hiba-Alili) [inline src/docs/admin/ProActiveAdminGuide.adoc:214]: Do you mean, "We currently provide support to "?

**Review by Hiba-Alili — APPROVED** [review]

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 3 above)
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc:214]: yes, I will fix it

**Pull request merged**